base/shell/explorer/settings.cpp

```diff
@@ -60,6 +61,9 @@ BOOL TaskbarSettings::Load()
     cbSize = sizeof(sr);
     dwRet = SHGetValueW(hkExplorer, L"StuckRects2", L"Settings", NULL, &sr, &cbSize);
 
+    dwRet = SHGetValueW(hkExplorer, L"Advanced", L"TaskbarSmallIcons", NULL, &dwValue, &cbSize);
```

Comment:
Huh, you need to move that SHGetValueW call before the StuckRects2 setting made above..
Because now you clobber the dwRet check done below that is related to the "StuckRects2" setting call.

Reply:
You're right, I moved this to not break the StuckRects2 setting call.